android/app/build.gradle

```diff
@@ -108,50 +107,46 @@ dependencies {
     implementation 'com.jakewharton:butterknife:8.6.0'
     implementation 'com.amulyakhare:com.amulyakhare.textdrawable:1.0.1'
     implementation 'com.facebook.stetho:stetho:1.5.0'
+    implementation 'com.github.jasminb:jsonapi-converter:0.7'
     implementation 'com.facebook.stetho:stetho-okhttp3:1.5.0'
     implementation 'com.uphyca:stetho_realm:2.1.0'
+
     implementation 'io.reactivex.rxjava2:rxandroid:2.0.1'
     implementation 'com.github.yalantis:ucrop:2.2.1'
     implementation 'io.reactivex.rxjava2:rxjava:2.1.3'
-    implementation "android.arch.lifecycle:reactivestreams:1.0.0"
+    implementation 'android.arch.lifecycle:reactivestreams:1.0.0'
     annotationProcessor 'com.jakewharton:butterknife-compiler:8.6.0'
-
+    kapt 'com.jakewharton:butterknife-compiler:8.6.0'
     // Googleplay Variant
     googleplayImplementation 'com.google.android.gms:play-services-maps:11.8.0'
     googleplayImplementation 'com.google.android.gms:play-services-location:11.8.0'
     googleplayImplementation 'com.google.maps.android:android-maps-utils:0.4'
-
     // Fdroid Variant
     fdroidImplementation 'org.osmdroid:osmdroid-android:5.1'
     fdroidImplementation 'org.apache.directory.studio:org.slf4j.api:1.7.2'
     fdroidImplementation 'org.slf4j:slf4j-simple:1.7.14'
-
     // Unit Test
     testImplementation 'junit:junit:4.12'
     testImplementation 'org.mockito:mockito-core:2.13.0'
     testImplementation 'org.threeten:threetenbp:1.3.6'
-
     //Instrumentation Test
     androidTestImplementation('com.android.support.test.espresso:espresso-core:3.0.1', {
         exclude group: 'com.android.support', module: 'support-annotations'
     })
-
     // Catch all the leaks
     debugImplementation 'com.squareup.leakcanary:leakcanary-android:1.5.4'
     releaseImplementation 'com.squareup.leakcanary:leakcanary-android-no-op:1.5.4'
     testImplementation 'com.squareup.leakcanary:leakcanary-android-no-op:1.5.4'
-
     //Branch integration for deep linking
-    implementation ('io.branch.sdk.android:library:2.5.9') {
+    implementation('io.branch.sdk.android:library:2.5.9') {
         exclude module: 'answers-shim'
     }
-
     //ViewModel and LiveData
-    implementation "android.arch.lifecycle:extensions:1.0.0"
-
+    implementation 'android.arch.lifecycle:extensions:1.0.0'
     //Zoomable image view
     compile 'com.github.chrisbanes:PhotoView:2.1.3'
     compile "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
+    implementation project(':data')
```

Comment:
Only this line needs to be changed in build.gradle

There are other unneeded changes

Reply:
No. I needed to move the JsonApi library and the realm dependency to the library module. So there are those changes.

Reply:
There are other changes as well, please see to it